build.sh

```diff
@@ -18,25 +17,26 @@ ARGS=$*
 # script, and that this script resides in the repo dir!
 REPODIR=$(cd $(dirname $0); pwd)
 
-VALIDARGS="clean libnvstrings nvstrings libcudf cudf dask_cudf benchmarks tests -v -g -n -l --allgpuarch --disable_nvtx --show_depr_warn -h"
-HELP="$0 [clean] [libcudf] [cudf] [dask_cudf] [benchmarks] [tests] [-v] [-g] [-n] [-h] [-l]
-   clean            - remove all existing build artifacts and configuration (start
-                      over)
-   libnvstrings     - build the nvstrings C++ code only
-   nvstrings        - build the nvstrings Python package
-   libcudf          - build the cudf C++ code only
-   cudf             - build the cudf Python package
-   dask_cudf        - build the dask_cudf Python package
-   benchmarks       - build benchmarks
-   tests            - build tests
-   -v               - verbose build mode
-   -g               - build for debug
-   -n               - no install step
-   -l               - build legacy tests
-   --allgpuarch     - build for all supported GPU architectures
-   --disable_nvtx   - disable inserting NVTX profiling ranges
-   --show_depr_warn - show cmake deprecation warnings
-   -h               - print this text
+VALIDARGS="clean libnvstrings nvstrings libcudf cudf dask_cudf benchmarks tests -v -g -n -l -libcudatasource --allgpuarch --disable_nvtx --show_depr_warn -h"
+HELP="$0 [clean] [libcudf] [cudf] [dask_cudf] [benchmarks] [tests] [-v] [-g] [-n] [-h] [-l] [-libcudatasource]
+   clean                - remove all existing build artifacts and configuration (start
+                          over)
+   libnvstrings         - build the nvstrings C++ code only
+   nvstrings            - build the nvstrings Python package
+   libcudf              - build the cudf C++ code only
+   cudf                 - build the cudf Python package
+   dask_cudf            - build the dask_cudf Python package
+   benchmarks           - build benchmarks
+   tests                - build tests
+   -v                   - verbose build mode
+   -g                   - build for debug
+   -n                   - no install step
+   -l                   - build legacy tests
+   -libcudatasource     - build the cudatasource C++ code only
```

Comment:
If we want this integrated into the build script it should be a separate build target, and we should name it say `libcudf_kafka` instead of `cuDataSource`.